modules/openid_connect/lib/open_project/openid_connect/engine.rb

```diff
@@ -60,22 +60,13 @@ class Engine < ::Rails::Engine
       end
     end
 
-    initializer "openid_connect.configure" do
-      ::Settings::Definition.add(
-        OpenProject::OpenIDConnect::CONFIG_KEY, default: {}, writable: false
-      )
-    end
-
-    initializer "openid_connect.form_post_method" do
```

Comment:
Is this safe to be removed?

Reply:
I'm not aware of a code path that sets this form_post, and it's a huge security risk